<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>angleMeasureDialog</class>
 <widget class="QWidget" name="angleMeasureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pulsars Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>530</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Angle Measure Plug-in Configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Angle Measure</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="showEquatorial_GroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Show in Equatorial Coordinates</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="equatorialHorizontalLayout">
            <item>
             <widget class="QCheckBox" name="showPositionAngleCheckBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Display with position angle</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="equatorialLineHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="equatorialTextColorToolButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="equatorialTextColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Text</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="equatorialLineColorToolButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="equatorialLineColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Line</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="showHorizontal_GroupBox">
         <property name="title">
          <string>Show in Horizontal Coordinates</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="5" column="0">
           <widget class="QCheckBox" name="azAltStartOnSkyCheckBox">
            <property name="text">
             <string>Attach start point to rotating sky</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="azAltEndOnSkyCheckBox">
            <property name="text">
             <string>Attach end point to rotating sky</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="showPositionAngleHorizontalCheckBox">
              <property name="toolTip">
               <string>(e.g. for Dobson starhopping)</string>
              </property>
              <property name="text">
               <string>Display with position angle </string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalLineHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="horizontalTextColorToolButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="horizontalTextColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Text</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="horizontalLineColorToolButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="horizontalLineColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Line</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="optionsBox">
         <property name="title">
          <string>Options</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="useDmsFormatCheckBox">
            <property name="text">
             <string>Use dms format for angles</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="followCursorCheckBox">
            <property name="toolTip">
             <string>When set, measurements are shown near the end of the line instead of its starting point.</string>
            </property>
            <property name="text">
             <string>Show measurements near mouse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="restoreDefaultsButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Restore defaults</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string comment="tab in plugin windows">About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="aboutTextBrowser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabs</tabstop>
  <tabstop>showEquatorial_GroupBox</tabstop>
  <tabstop>showPositionAngleCheckBox</tabstop>
  <tabstop>equatorialTextColorToolButton</tabstop>
  <tabstop>equatorialLineColorToolButton</tabstop>
  <tabstop>showHorizontal_GroupBox</tabstop>
  <tabstop>showPositionAngleHorizontalCheckBox</tabstop>
  <tabstop>horizontalTextColorToolButton</tabstop>
  <tabstop>horizontalLineColorToolButton</tabstop>
  <tabstop>azAltStartOnSkyCheckBox</tabstop>
  <tabstop>azAltEndOnSkyCheckBox</tabstop>
  <tabstop>useDmsFormatCheckBox</tabstop>
  <tabstop>followCursorCheckBox</tabstop>
  <tabstop>restoreDefaultsButton</tabstop>
  <tabstop>aboutTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
